tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -102,35 +104,48 @@ func (c *cluster) bootstrap() error {
 		}
 	}
 
-	// TODO: Too intensive stressers can panic etcd member with
-	// 'out of memory' error. Put rate limits in server side.
-	stressN := 100
 	c.Stressers = make([]Stresser, len(members))
-	limiter := rate.NewLimiter(rate.Limit(c.stressQPS), c.stressQPS)
 	for i, m := range members {
+		c.Stressers[i] = c.createStresser(m)
+		go c.Stressers[i].Stress()
+	}
+
+	c.Size = size
+	c.Members = members
+	return nil
+}
+
+func (c *cluster) createStresser(m *member) Stresser {
+	switch c.stresserType {
+	case "nop":
+		return &nopStresser{}
+	case "default":
+		// TODO: Too intensive stressers can panic etcd member with
+		// 'out of memory' error. Put rate limits in server side.
+		stressN := 100
+
 		if c.v2Only {
-			c.Stressers[i] = &stresserV2{
+			return &stresserV2{
 				Endpoint:       m.ClientURL,
 				keySize:        c.stressKeySize,
 				keySuffixRange: c.stressKeySuffixRange,
 				N:              stressN,
 			}
 		} else {
-			c.Stressers[i] = &stresser{
+			return &stresser{
 				Endpoint:       m.grpcAddr(),
 				keyLargeSize:   c.stressKeyLargeSize,
 				keySize:        c.stressKeySize,
 				keySuffixRange: c.stressKeySuffixRange,
 				N:              stressN,
-				rateLimiter:    limiter,
+				rateLimiter:    rate.NewLimiter(rate.Limit(c.stressQPS), c.stressQPS),
```

Comment:
Ah, this is incorrect as you pointed. I'll revert the part in the next update. Thanks.